src/renderer/components/dialogs/CreateChat.tsx

```diff
@@ -339,12 +344,16 @@ export function AddMemberInnerDialog({
     onOk(addMembers)
   }
 
+  const inputRef = useRef(null)
+  useLayoutEffect(() => inputRef?.current?.focus(), [contactsNotInGroup])
+
```

Comment:
what does this do?

Reply:
It keeps the search bar focused in the addmember dialog